### PR TITLE
[DNM] Adds RNG Disruption wizard event

### PR DESCRIPTION
**Pull request by Iamgoofball**
rng defaults to 1 for 30 seconds

please don't merge this just yet i need to move it to a new file and test it on a non-linux computer

### CODE REVIEW

File: code/modules/events/wizard/lava.dm [hunk 1]
```diff
@@ -39,4 +39,28 @@
 			F.desc = initial(F.desc)
 			F.overlays.Cut()
 			F.lava = 0
-			F.update_icon()
\ No newline at end of file
+			F.update_icon()
+
+var/global/disable_rng = 0
+
+prob()
```
Comment (JJRcop) [inline code/modules/events/wizard/lava.dm]: I think you have an indentation problem or you're not committing something.

**Comment by RemieRichards** [comment]
this literally disables rng...

oh dear.

**Comment by paprka** [comment]
the bad ideas are just fresh off the boat huh

**Comment by Tenebrosity** [comment]
It's a neat idea, but it's probably going to screw everything up.

**Comment by AlexanderUlanH** [comment]
This will make explosions, EMP's, genetics, chems, viruses, and all sorts of other things silly.  Unless it breaks something important, :+1:

**Comment by dannno** [comment]
what in the world did you think was going to happen

**Comment by GunHog** [comment]
Goof, you will need to test almost <b>every instance of prob()</b> to allow something like this. This has a *massive* potential of ripping the game to shreds. I am not saying we should discard the idea, I just recommend extensive testing of this feature to ensure it does not do anything server-breaking.

**Comment by CollenN** [comment]
Goofball, do you even have a brain?

**Comment by Iamgoofball** [comment]
:^(

**Comment by RemieRichards** [comment]
Well he's either below our level of intelligence <i>or somehow above...</i>

**Comment by Kraseo** [comment]
the fedora has sapped him of his own free will

now he has found his final purpose

to break SS13 in the worst way mankind has ever thought of

**Comment by AlexanderUlanH** [comment]
Well think of it this way.  If every instance of prob() coming up true breaks something, it's a bug waiting to happen, so we should merge it so we can go ahead and find it before it happens by chance.

**Comment by Miauw62** [comment]
every baby is a mass murderer waiting to happen, so we should kill them all

**Pull request closed without merging**